common/djangoapps/student/models.py

```diff
@@ -59,6 +59,26 @@
 AUDIT_LOG = logging.getLogger("audit")
 SessionStore = import_module(settings.SESSION_ENGINE).SessionStore  # pylint: disable=invalid-name
 
+UNENROLLED_TO_ALLOWEDTOENROLL = _('from unenrolled to allowed to enroll')
+ALLOWEDTOENROLL_TO_ENROLLED = _('from allowed to enroll to enrolled')
+ENROLLED_TO_ENROLLED = _('from enrolled to enrolled')
+ENROLLED_TO_UNENROLLED = _('from enrolled to unenrolled')
+UNENROLLED_TO_ENROLLED = _('from unenrolled to enrolled')
+ALLOWEDTOENROLL_TO_UNENROLLED = _('from allowed to enroll to enrolled')
+UNENROLLED_TO_UNENROLLED = _('from unenrolled to unenrolled')
```

Comment:
It doesn't seem like a good idea to translate these strings, because it looks like we're never going to be exposing them to users.

If we definitely need these translations, I think these strings will be confusing to translators so I would suggest putting in translators' notes: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#python-source-code